.github/workflows/push.yml

```diff
@@ -53,90 +53,37 @@ jobs:
-      - name: Test Tracing JIT
-        uses: ./.github/actions/test-linux
-        with:
-          testArtifacts: ${{ matrix.debug && 'DEBUG' || 'RELEASE' }}_${{ matrix.zts && 'ZTS' || 'NTS' }} Tracing JIT
-          runTestsParameters: >-
-            -d zend_extension=opcache.so
-            -d opcache.enable_cli=1
-            -d opcache.jit_buffer_size=16M
-      - name: Verify generated files are up to date
-        uses: ./.github/actions/verify-generated-files
-  LINUX_X32:
-    name: LINUX_X32_DEBUG_ZTS
-    runs-on: ubuntu-latest
-    container:
-      image: ubuntu:20.04
-    env:
-      MYSQL_TEST_HOST: mysql
-      PDO_MYSQL_TEST_DSN: mysql:host=mysql;dbname=test
-      PDO_MYSQL_TEST_HOST: mysql
-    services:
-      mysql:
-        image: mysql:8
-        ports:
-          - 3306:3306
-        env:
-          MYSQL_DATABASE: test
-          MYSQL_ROOT_PASSWORD: root
-    steps:
-      - name: git checkout
-        uses: actions/checkout@v3
-      - name: apt
-        uses: ./.github/actions/apt-x32
-      - name: ./configure
-        uses: ./.github/actions/configure-x32
-        with:
-          configurationParameters: >-
-            --enable-debug
-            --enable-zts
-      - name: make
-        run: make -j$(/usr/bin/nproc) >/dev/null
-      - name: make install
-        uses: ./.github/actions/install-linux-x32
-      - name: Test
-        uses: ./.github/actions/test-linux
-      - name: Test Tracing JIT
-        uses: ./.github/actions/test-linux
-        with:
-          runTestsParameters: >-
-            -d zend_extension=opcache.so
-            -d opcache.enable_cli=1
-            -d opcache.jit_buffer_size=16M
-  MACOS_DEBUG_NTS:
-    runs-on: macos-11
-    steps:
-      - name: git checkout
-        uses: actions/checkout@v3
-      - name: brew
-        uses: ./.github/actions/brew
-      - name: ./configure
-        uses: ./.github/actions/configure-macos
-        with:
-          configurationParameters: --enable-debug --disable-zts
-      - name: make
-        run: |-
-          export PATH="/usr/local/opt/bison/bin:$PATH"
-          make -j$(sysctl -n hw.logicalcpu) >/dev/null
-      - name: make install
-        run: sudo make install
-      - name: Test
-        uses: ./.github/actions/test-macos
-        with:
-          testArtifacts: ${{ matrix.debug && 'DEBUG' || 'RELEASE' }}_${{ matrix.zts && 'ZTS' || 'NTS' }}
-      - name: Test Tracing JIT
-        uses: ./.github/actions/test-macos
-        with:
-          testArtifacts: ${{ matrix.debug && 'DEBUG' || 'RELEASE' }}_${{ matrix.zts && 'ZTS' || 'NTS' }} Tracing JIT
-          runTestsParameters: >-
-            -d zend_extension=opcache.so
-            -d opcache.enable_cli=1
-            -d opcache.protect_memory=1
-            -d opcache.jit_buffer_size=16M
-      - name: Verify generated files are up to date
-        uses: ./.github/actions/verify-generated-files
+      - name: Test Drupal
```

Comment:
You should move this to a new job for testing analogous to https://github.com/php/php-src/pull/9942. The binary needs to be built with the sanitize flags for this to be meaningful.

Reply:
Yes,I will 

ATM trying to connect to pgsql but it looks broken

Reply:
I would just add another service like here: https://github.com/php/php-src/blob/master/.github/workflows/nightly.yml#L130-L137

Reply:
Or just use the existing mysql db if you can.

Reply:
Somehow pgsql connection has issues, I limited tests to sqlite and mysql now